test/grub2_widgets_test.rb

```diff
@@ -623,9 +623,9 @@ def stub_widget_value(id, value)
 
   it "enlists all available sections" do
     expect(subject.items).to eq([
```

Comment:
I think if you move the opening `[` on a separate line then Rubocop does not indent that much...